Docker/docker-compose.yml

```diff
@@ -23,5 +23,5 @@ services:
       - walletd-data-volume:/opt/eos/bin/data-dir
 
 volumes:
-  eosd-data-volume:
-  walletd-data-volume:
\ No newline at end of file
+  eosd-data-volume: /data/store/eos
+  walletd-data-volume: /data/store/wallet
```

Comment:
We prefer not to hard code the mount volumes as this can cause issues if not writable by the user. You can always use docker-compose.override.yml to change the default if a user desires to do so.